packages/mephisto-task/src/utils.js

```diff
@@ -93,23 +93,8 @@ export function postProviderRequest(endpoint, data) {
   return postData(url, { provider_data: data });
 }
 
-export function requestAgent(mephisto_worker_id) {
-  return postProviderRequest(
-    "/request_agent",
-    getAgentRegistration(mephisto_worker_id)
-  );
-}
-
-export function registerWorker() {
-  return postProviderRequest("/register_worker", getWorkerRegistrationInfo());
-}
-
-// Sends a request to get the initial task data
-export function getInitTaskData(mephisto_worker_id, agent_id) {
-  return postProviderRequest("/initial_task_data", {
-    mephisto_worker_id: mephisto_worker_id,
-    agent_id: agent_id,
-  });
+export function requestAgent() {
+  return postProviderRequest("/request_agent", getAgentRegistration());
```

Comment:
Nice. I appreciate this single handshake, and the simplification over the multiple handshakes that needed to occur before.